packages/iris-grid/src/mousehandlers/IrisGridTooltipMouseHandler.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  type EventHandlerResult,
+  type Grid,
+  type GridPoint,
+  isLinkToken,
+  GridTooltipMouseHandler,
+  type GridRangeIndex,
+} from '@deephaven/grid';
+import deepEqual from 'fast-deep-equal';
+import type IrisGrid from '../IrisGrid';
+
+class IrisGridTooltipMouseHandler extends GridTooltipMouseHandler {
+  private irisGrid: IrisGrid;
+
+  private lastColumn: GridRangeIndex;
+
+  private lastRow: GridRangeIndex;
+
+  constructor(irisGrid: IrisGrid) {
+    super();
+
+    this.irisGrid = irisGrid;
+    this.lastColumn = null;
+    this.lastRow = null;
+  }
+
+  private destroyTooltip(): void {
+    this.irisGrid.setState({ hoverTooltipProps: null });
+  }
+
+  protected setCursor(gridPoint: GridPoint, grid: Grid): EventHandlerResult {
+    if (this.isHoveringLink(gridPoint, grid)) {
+      this.cursor = 'pointer';
+      return { stopPropagation: false, preventDefault: false };
+    }
+    this.cursor = null;
+    return false;
+  }
+
+  onMove(gridPoint: GridPoint, grid: Grid): EventHandlerResult {
+    const { model } = this.irisGrid.props;
+    const isUserHoveringLink = this.isHoveringLink(gridPoint, grid);
+    const dateTooltip = model.tooltipForCell(gridPoint.column, gridPoint.row);
```

Comment:
Should just call this `tooltip`. Nothing here suggests it would be a date only, that's up to the model implementation details.